aspnetcore/migration/21-to-22.md

```diff
@@ -68,6 +68,8 @@ The metapackage reference should resemble the following `<PackageReference />` n
 </ItemGroup>
 ```
 
+When targeting .NET Core and transitively referencing a .NET Core project in another .NET Core project (such as a unit test project), implicit references will map to 2.1 assemblies, resulting in build errors.  In this case, add `<PackageReference Include="Microsoft.AspNetCore.App" />` as a package reference in your project (e.g., the unit test project referencing ASP.NET Core metapackage).
```

Comment:
I still don't think implicit references will map to 2.1 assemblies.  I would expect that you just wouldn't get compile-time references to ASP.NET at all.

What is the behavior you're seeing that makes you think 2.1 assemblies are involved?

Reply:
Pivoted the Visual Studio Errors window copied values into a format more readable on GitHub (wish this is how Visual Studio copying of this grid worked by default!)

```
Severity: Error		
Code: CS1705
Description: Assembly '.WebApi' with identity 'WebApi, Version=1.5.0.0, Culture=neutral, PublicKeyToken=null' uses 'Microsoft.AspNetCore.Mvc.ViewFeatures, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' which has a higher version than referenced assembly 'Microsoft.AspNetCore.Mvc.ViewFeatures' with identity 'Microsoft.AspNetCore.Mvc.ViewFeatures, Version=2.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'
Project: WebApi.Tests
File: D:\source\John.Zabroski\Source\WebApi.Tests\CSC
Line:	 1
Suppression State: Active
```

This happens even if I clean bin and obj directories.  It also happens even if I add the implicit package reference in Tests, build successfully, remove it without cleaning bin and obj directories, and build again.